src/SMTLIB/Backends.hs

```diff
@@ -77,8 +77,9 @@ initSolver ::
   Backend ->
   -- | whether to enable lazy mode (see 'Solver' for the meaning of this flag)
   Bool ->
-  -- | function for logging the solver's activity
-  (Builder -> IO ()) ->
+  -- | function for logging the solver's activity;
+  -- it should manually jump lines if needed
```

Comment:
Yeah I found it surprisingly hard to say that in a concise way, because "the logging function" references the function passed as an argument, not how we actually it. Any suggestions ?